db/migrate/20181026104718_create_suggest_a_registers.rb

```diff
@@ -0,0 +1,7 @@
+class CreateSuggestARegisters < ActiveRecord::Migration[5.2]
+#   def change
```

Comment:
Is this still needed?

Reply:
Doesn't look like it - I've removed it and seems to deploy okay on staging.